Makefile

```diff
@@ -67,23 +63,16 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
-
-docker-push: ## Push docker image with the manager.
-	docker push ${IMG}
```

Comment:
Do we need an equivalent `ko` command to push the image here?

Reply:
https://github.com/k8s-service-bindings/implementation/pull/19#discussion_r692268506